src/protocols/secure_channel/PairingSession.h

```diff
@@ -174,6 +163,29 @@ class DLL_EXPORT PairingSession : public SessionDelegate
         else
         {
             err = OnFailureStatusReport(report.GetGeneralCode(), report.GetProtocolCode());
+
+            if (report.GetGeneralCode() == Protocols::SecureChannel::GeneralStatusCode::kBusy &&
+                report.GetProtocolCode() == Protocols::SecureChannel::kProtocolCodeBusy)
+            {
+                uint16_t minimumWaitTime = 0;
+                if (!report.GetProtocolData().IsNull() && report.GetProtocolData()->DataLength() >= sizeof(minimumWaitTime))
```

Comment:
You could just skip the length check here; Read16 will fail on insufficient length, and that will be fine (and will helpfully log that the data was malformed).